src/data/reducers/index.js

```diff
@@ -9,6 +9,7 @@ import csv from './csv';
 import userProfile from './userProfile';
 import coupons from './coupons';
 import sidebar from './sidebar';
+import codeAssignmentReducer from './codeAssignment';
```

Comment:
This should be named just `codeAssignment`. It uses this name as a key in the Redux store object.